kolla/defaults.yml

```diff
@@ -196,7 +205,7 @@ enable_jupyterhub_external_frontend: "{{ enable_jupyterhub_external }}"
 # Neutron
 enable_neutron: yes
 enable_openvswitch: yes
-enable_neutron_fwaas: yes
+enable_neutron_fwaas: no
```

Comment:
is `fwaas` supported anymore? I don't think it has a xena release. If not, can this be removed entirely?